src/web/mjs/connectors/templates/MojoPortalComic.mjs

```diff
@@ -34,17 +42,17 @@ export default class MojoPortalComic extends Connector {
 
     async _getMangaFromURI(uri) {
         let request = new Request(uri, this.requestOptions);
-        let data = await this.fetchDOM(request, 'meta[property="og:title"]');
+        let data = await this.fetchDOM(request, this.queryMangaTitle);
         let id = uri.pathname + uri.search;
-        let title = data[0].content.replace(new RegExp('-? ' + this.label, 'i'), '').trim();
+        let title = (data[0].content || data[0].textContent).replace(new RegExp('-? ' + this.label, 'i'), '').trim();
         return new Manga(this, id, title);
     }
 
     async _getMangas() {
         let mangaList = [];
-        let request = new Request(this.url + '/?page=', this.requestOptions);
-        let data = await this.fetchDOM(request, 'div.pagination-outter ul.pagination li:last-of-type a');
-        let pageCount = parseInt(data[0] ? data[0].href.match(/\d+$/)[0] : 1);
+        let request = new Request(this.url + this.path, this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryMangasPageCount);
+        let pageCount = parseInt(data[0].href.match(this.pathMatch)[1]);
```

Comment:
Previous code handled the use case when no pagination element was found `data[0] ? ... : 1` is it safe to remove this check?

Reply:
all the connectors that use _getMangas from the conector still are functional